Makefile

```diff
@@ -0,0 +1,50 @@
+KUTTL_VERSION=0.4.0
+KIND_VERSION=0.8.1
+KUBERNETES_VERSION ?= 1.17.5
+KUBECONFIG=kubeconfig
+
+OS=$(shell uname -s | tr '[:upper:]' '[:lower:]')
+MACHINE=$(shell uname -m)
+KIND_MACHINE=$(shell uname -m)
+ifeq "$(KIND_MACHINE)" "x86_64"
+  KIND_MACHINE=amd64
+endif
+
+export PATH := $(shell pwd)/bin/:$(PATH)
+
+ARTIFACTS=dist
+
+bin/kind_$(KIND_VERSION):
+	mkdir -p bin/
+	curl -Lo bin/kind_$(KIND_VERSION) https://github.com/kubernetes-sigs/kind/releases/download/v$(KIND_VERSION)/kind-$(OS)-$(KIND_MACHINE)
+	chmod +x bin/kind_$(KIND_VERSION)
+
```

Comment:
inside the repository, it will only pull if the binary doesn't exists inside `./bin/` 
So consequent builds don't always pull the same binary again and again in the local environment. So it does cache them and will fetch again only if the version is changed or `bin` directory is deleted.